deploy/helm/sumologic/Chart.yaml

```diff
@@ -1,5 +1,12 @@
 apiVersion: v1
-appVersion: "0.12.0"
-description: A Helm chart for collecting Kubernetes logs, metrics and events into Sumo Logic.
 name: sumologic
 version: 0.12.0
+appVersion: 0.12.0
+description: A Helm chart for collecting Kubernetes logs, metrics and events into Sumo Logic.
+keywords:
+  - monitoring
+  - logging
+icon: https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/images/sumo_logic_logo.png
```

Comment:
hmm this gives me a 404 if I open in a browser

Reply:
Yes, the image is part of this PR. It will be in master branch after merging this PR and then the link will work.